files/scripts/swss.sh

```diff
@@ -149,15 +151,17 @@ start() {
         clean_up_tables STATE_DB "'PORT_TABLE*', 'MGMT_PORT_TABLE*', 'VLAN_TABLE*', 'VLAN_MEMBER_TABLE*', 'LAG_TABLE*', 'LAG_MEMBER_TABLE*', 'INTERFACE_TABLE*', 'MIRROR_SESSION*', 'VRF_TABLE*', 'FDB_TABLE*', 'FG_ROUTE_TABLE*', 'BUFFER_POOL*', 'BUFFER_PROFILE*', 'MUX_CABLE_TABLE*'"
     fi
 
-    # start service docker
-    /usr/bin/${SERVICE}.sh start $DEV
-    debug "Started ${SERVICE}$DEV service..."
+    # For fixed platforms start swss service here
```

Comment:
It would be good to keep the original code here ( and other scripts), and add call to check_asic_status() here + also in the wait() call below. 
In check_asic_status() , if it is modular chassis platform, do check for ASIC presence, else nop.

Reply:
Have made changes. Please review if this is what you have in mind. I have to still provide an argument to indicate whether this is being called as part of start() or wait().